service/dynamodb/expression/operand.go

```diff
@@ -51,6 +52,28 @@ type KeyBuilder struct {
 	key string
 }
 
+// setValueMode will specify the type of SetValueBuilder. The default value will
+// be unsetValue so if a User were to create an empty SetValueBuilder, we can
+// return ErrUnsetSetValue when BuildOperand() is called.
+type setValueMode int
+
+const (
+	unsetValue setValueMode = iota
+	plus
+	minus
+	listAppend
+	ifNotExists
```

Comment:
Nit it would be helpful for this to be suffixed with `ValueMode` to keep these constants clear